.travis.sh

```diff
@@ -28,6 +28,15 @@ apt-get update -qq && apt-get install -qq -y -q wget sudo lsb-release gnupg # fo
 travis_time_start setup.before_install
 #before_install:
 # Define some config vars.
+# Setup apt target
+if [ "$OS_NAME" = "debian" ]; then
+  # Using httpredir here to match mirror used in osrf image
+  # (https://github.com/osrf/multiarch-docker-image-generation/blob/d251b9a/build-image.sh#L46)
+  echo deb http://httpredir.debian.org/debian $OS_CODE_NAME contrib non-free | tee -a /etc/apt/sources.list
+  # Hit cloudfront mirror because of corrupted packages on fastly mirrors (https://github.com/ros-infrastructure/ros_buildfarm/issues/455)
+  # You can remove this line to target the default mirror or replace this to use the mirror of your preference
+  sed -i 's/httpredir.debian.org/cloudfront.debian.net/' /etc/apt/sources.list
```

Comment:
Thank you!
Yeah, it seems not work well on Travis. Maybe I should just restart the build when we encounter the apt fail.

```
E: Failed to fetch http://cloudfront.debian.net/debian/pool/main/k/keyrings.alt/python-keyrings.alt_1.3-1_all.deb  503  Service Temporarily Unavailable [IP: 52.84.64.254 80]
E: Failed to fetch http://cloudfront.debian.net/debian/pool/main/l/lazr.uri/python-lazr.uri_1.0.3-2_all.deb  503  Service Temporarily Unavailable [IP: 52.84.64.254 80]
E: Failed to fetch http://cloudfront.debian.net/debian/pool/main/s/simplejson/python-simplejson_3.10.0-1_amd64.deb  503  Service Temporarily Unavailable [IP: 52.84.64.254 80]
E: Failed to fetch http://cloudfront.debian.net/debian/pool/main/p/python-wadllib/python-wadllib_1.3.2-3_all.deb  503  Service Temporarily Unavailable [IP: 52.84.64.254 80]
E: Failed to fetch http://cloudfront.debian.net/debian/pool/main/z/zope.interface/python-zope.interface_4.3.2-1_amd64.deb  503  Service Temporarily Unavailable [IP: 52.84.64.254 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```